### DISCUSSION TITLE
Controller not connecting over bluetooth

### DISCUSSION THREAD

**Discussion by ghost** [nefarius/DsHidMini#212] [Q&A]
Hey guys
So I installed the DsHidMini on my pc and it's not working over bluetooth. I've tried everything I can. It works normally with the wired connection though, just can't get the controller to connect over bluetooth. Please help

**Comment by DJm00n** [comment]
Which controller, which bluetooth dongle, which system? It's impossible to answer your question without any meaningful details.

**Reply by ghost** [reply]
I'm using the Dual Shock 3 Controller, bluetooth device is the Qualcomm Atheros AR9462 and using a Probook 640

**Reply by DJm00n** [reply]
For DS3 with bluetooth support you need to install https://github.com/ViGEm/BthPS3

**Reply by ghost** [reply]
I had installed it previously before the main DsHidMini installation. I also tried adding the controller name in BthPS3 in Registry Editor like someone pointed out in one other discussion and still no luck

**Reply by DJm00n** [reply]
Try to follow this guide to configure your DS3: https://vigem.org/projects/DsHidMini/How-to-Install/
It should pair your controller automatically after connecting it via USB. With your controller connected by USB, open the DsHidMini Control Utility (the DSHMC.exe file that comes along the driver archive) to see if the controller gets detected.

**Reply by DJm00n** [reply]
You can also check and delete old\other gamepad drivers\emulators via [DriverStoreExplorer](https://github.com/lostindark/DriverStoreExplorer) tool.

**Reply by DJm00n** [reply]
Also ensure that your Bluetooth host device\dongle is supported: https://github.com/ViGEm/BthPS3#supported-bluetooth-host-devices
